.github/workflows/periodic.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     uses: ./.github/workflows/_linux-build.yml
     needs: get-label-type
     with:
-      runner: "${{ needs.get-label-type.outputs.label-type }}amz2023.linux.2xlarge"
+      runner_prefix: "${{ needs.get-label-type.outputs.label-type }}amz2023."
```

Comment:
Why repeated `needs.get-label-type.outputs.label-type` can not be refactored away?

Reply:
how would one do that?  That prefix today contains either the "lf." prefix or an empty string and is set by the get-label-type job to figure out if the jobs should run in the LF fleet or Meta fleet